src/Spectre.AngularClient/src/app/divik-results/shared/divik.service.spec.ts

```diff
@@ -47,4 +78,108 @@ describe('DivikService', () => {
         expect(divik.data[0][2]).toEqual(9.99);
       });
     }));
+
+    it('parses max k from DiviK config', inject([DivikService, MockBackend],
```

Comment:
I am not sure, but I suppose we can still repeat even less code. The only 2 things change between this test and those below: config field tested and its expected value. Are we able to reduce amount of repeated code further?